src/utils/global/tooltip/tooltip.module.scss

```diff
@@ -0,0 +1,39 @@
+@import '../../../variables.scss';
+
+.sharedStyles {
+  background-color: $tooltip-background-color;
+  border-radius: 0.5rem;
+  color: white;
+  font-size: 1rem;
+  left: calc(100% + .75rem);
+  padding: 0.5rem;
+  position: absolute;
+  transition: $transition;
+
+}
+
+.triangle {
+  position: absolute;
+  margin-top: -5px;
+  right: 100%;
+  top: 50%;
+  border-width: 5px;
+  border-style: solid;
+  border-color: transparent $tooltip-background-color transparent transparent;
+}
+
+.children {
+  position: relative;
+  display: flex;
+  height: 100%;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+}
+.wrapper {
+  display: inline-flex;
+  height: 100%;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
```

Comment:
```suggestion
  position: relative;
```

I'm not sure any of this should be default styling really -- I think the only thing I'd expect here is a `position: relative` to ensure the tooltip is positioned correctly.

I would expect the Tooltip to accept a className prop though, which gets applied to the wrapper -- this would allow specific uses to pass styles as needed, such as width/height 100%.

Reply:
I'm realising that using `position: relative` might cause some issues with the conservation status tooltip, as it then forces it to be positioned within the scrollable container -- off the top of top of my head I can't think of a great solution to this.

Currently, the position completely breaks when the card starts scrolling, but with `position: relative` it will be broken in a different way.

![image](https://github.com/alveusgg/extension/assets/12371363/fda5dcb0-07fa-41e5-8076-12b4f5a341dc)

Reply:
I'm not sure if there's a good CSS-only solution to this, so might need a bit of JavaScript?

https://css-tricks.com/popping-hidden-overflow/ goes over a very similar problem and lands on JavaScript as the solution.